frontend/express/public/core/events/javascripts/countly.details.views.js

```diff
@@ -262,42 +302,45 @@
             lineLegend: function() {
                 return this.$store.getters["countlyAllEvents/legendData"];
             },
-            allEvents: function() {
-                var list = this.$store.getters["countlyAllEvents/allEvents"].list;
-                if (list) {
-                    return list.map(function(item) {
-                        return {
-                            "label": item,
-                            "value": item
-                        };
-                    });
-                }
-                return list;
-            },
             searchPlaceholder: function() {
-                var list = this.$store.getters["countlyAllEvents/allEvents"].list;
+                var list = this.$store.getters["countlyAllEvents/allEventsList"];
                 if (list) {
                     return this.i18n("events.all.search.placeholder", list.length);
                 }
                 return this.i18n("events.all.search.placeholder");
+            },
+            allEvents: function() {
+                return this.$store.getters["countlyAllEvents/allEventsList"];
+            },
+            eventDescription: function() {
+                return this.$store.getters["countlyAllEvents/allEventsProcessed"].eventDescription;
+            },
+            labels: function() {
+                return this.$store.getters["countlyAllEvents/allEventsProcessed"].chartDP;
+            },
+            limitAlerts: function() {
+                return this.$store.getters["countlyAllEvents/limitAlerts"];
             }
 
         },
         data: function() {
-            return {
-                description: CV.i18n('events.all.title.new')
-            };
+            return {description: CV.i18n('events.all.title.new') };
         },
         beforeCreate: function() {
             countlyCommon.setPeriod("30days");
             if (this.$route.params) {
                 this.$store.dispatch('countlyAllEvents/fetchSelectedEventName', this.$route.params.eventKey);
```

Comment:
you should use localStorage here.
var currEvent = this.$route.params.eventKey || localStorage.getItem("eventKey");
this.$store.dispatch('countlyAllEvents/fetchSelectedEventName', currEvent);

Reply:
as per our discussion this is expected in the upcoming pr.
keeping a note.